etc/scripts/json2csv.py

```diff
@@ -163,7 +163,7 @@ def flatten_scan(scan):
                             pack[nk] = '\n'.join(val)
                         if k == 'asserted_licenses':
                             # All license names are to be joined in a multi-line cell
-                            licenses = [license_info['license'] for license_info in val]
+                            licenses = [license_info.get('license', '') or '' for license_info in val]
```

Comment:
My first modification was to use `license_info.get('license', '')`, but this returns `None` instead of an empty string when `license` is `null`. I had to use the or to use the empty string when `None` was returned.